src/neuroconv/utils/json_schema.py

```diff
@@ -232,6 +233,18 @@ def fill_defaults(schema: dict[str, Any], defaults: dict[str, Any], overwrite: b
     if properties_reference not in schema and "patternProperties" in schema:
         properties_reference = "patternProperties"
 
+    # Handle additionalProperties case
```

Comment:
What's going on here? Why do you need to edit the json_schema?

Reply:
I was debugging something. I removed this now.